Introduction/tutorials/Introduction.md

```diff
@@ -1,106 +1,404 @@
+
+4. Click on the `get output` button to find a menu similar to this:
 
-## Subpart 2
+ ![](../images/101_07.png)
 
-Short introduction about this subpart.
+ Make sure that **Create one BED per** is set to `Whole Gene` (Whole Gene here really means Whole Feature), and click on **Send Query to Galaxy**. You will get your second item in your analysis history.
+
+5. Now **rename** your new dataset to `SNPs` so we can easily remember what the file contains.
+
+# Finding Exons with the highest number of SNPs
+
+Let’s remind ourselves that our objective was to find which exon contains the most SNPs. This  first step in answering this question will be joining the file with the exon locations with the file containing the SNP locations (here `join` is just a fancy word for printing the SNPs and exons that overlap side-by-side).
 
 :pencil2: ***Hands on!***
 
-1. First step
-2. Second step
-3. Third step
+1. Different Galaxy servers may have tools available under different sections, therefore it is often useful to use the **search bar** at the top of the tool panel to find your tool. Enter the word `join` in the search bar of the tool panel, and select the tool named `Join - the intervals of two datasets side-by-side`
+
+2. Select your file with exons as the first file, and the file with SNPs as the second file, and make sure **return** is set to `INNER JOIN` so that only matches are included in the output (i.e. only exons with SNPs in it and only SNPs that fall in exons)
 
-Some blabla
+ ![](../images/101_11.png)
+
+ **Note:** if you scroll down on this page, you will find an explanation of the tool.
+
+3. Click the **Execute** button and view the resulting file (with the eye icon). If everything went okay, you should see a file that looks similar to this:
+
+ ![](../images/101_joined.png)
+
+ Remember that variations are possible due to using different versions of UCSC databases, as long as you have similar looking columns you did everything right :)
+
+Let’s take a look at this dataset. The first six columns correspond to the exons, and the last six columns correspond to the SNPs. Column 4 contains the exon IDs, and column 10 contains the SNP IDs. In our screenshot you see that the first 5 lines in the file all have the same exon ID (`uc010gqp.2_cds_10_0_chr22_16287254_r`) but different SNP IDs, this means these lines represent 5 different SNPs that all overlapped with the same exon. So we can find the total number of SNPs in an exon simply by counting the number of lines that have the exon ID in the fourth column.
+
+:question: Question: for the first 3 exons in your file, what is the number of SNPs that fall into that exon?
```

Comment:
to me it looked a bit weird without, as if it maybe would not be clear they needed to answer this? but I do not care really strongly either way :)